jquery.flot.touchNavigate.js

```diff
@@ -72,31 +74,90 @@ can set the default in the options. */
             }
         }
 
+        function isPinchEvent(e) {
+            return e.touches && e.touches.length === 2;
+        }
+
         function onDragStart(e) {
             e.stopPropagation();
             e.preventDefault();
-            plot.getPlaceholder().css('cursor', plot.getOptions().pan.cursor);
-            startPageX = e.touches[0].pageX || e.touches[0].clientX;
-            startPageY = e.touches[0].pageY || e.touches[0].clientY;
-            plotState = plot.navigationState();
+            scaling = isPinchEvent(e);
+
+            if (!isPinchEvent(e)) {
+                scaling = false;
 
-            saveNavigationData(plot, e);
+                plot.getPlaceholder().css('cursor', plot.getOptions().pan.cursor);
+                startPageX = e.touches[0].pageX || e.touches[0].clientX;
+                startPageY = e.touches[0].pageY || e.touches[0].clientY;
```

Comment:
Mixing page and client coordinates looks strange and it has to be clarified. The ```pageX || clientX``` is all over the place. Worst case it should be moved into a function.